packages/thirdweb/src/wallets/smart/lib/constants.ts

```diff
@@ -6,6 +6,9 @@ export const DEBUG = false;
 export const DUMMY_SIGNATURE =
   "0xfffffffffffffffffffffffffffffff0000000000000000000000000000000007aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa1c";
 
+export const DEFAULT_ACCOUNT_FACTORY =
```

Comment:
how come we don't predict this instead?

Reply:
i guess we could, but need to pin it to a version because we don't want accounts switching on a new update of the factory. Felt safer to hardcode, and saves a roundtrip

Reply:
fair, what happens on chains where its not supported?

Reply:
I guess for now we need to pre-deploy on all `aa` supported chains?

Reply:
yeah, if the factory is not deployed (or you pass a random address) it fails to connect with:

`Failed to get account address with factory contract 0x564cf6453a1b0FF8DB603E92EA4BbD410dea45F3 on chain ID 137. Are you on the right chain?`